app/routes/crate/index.js

```diff
@@ -6,60 +6,61 @@ import Crate from 'cargo/models/crate';
 export default Ember.Route.extend({
     title: Ember.computed.reads("controller.name"),
 
-    setupController: function(controller, data) {
+    setupController(controller, data) {
         if (data instanceof Crate) {
             data = {crate: data, version: null};
         } else if (data instanceof Version) {
             data = {crate: data.get('crate'), version: data.get('num')};
         }
-        var self = this;
+
         this._super(controller, data.crate);
+
         controller.set('showAllVersions', false);
         controller.set('fetchingDownloads', true);
         controller.set('fetchingFollowing', true);
 
-        data.crate.get('keywords').then(function(keywords) {
-            controller.set('keywords', keywords);
-        });
+        data.crate.get('keywords').then((keywords) => controller.set('keywords', keywords));
 
         if (this.session.get('currentUser')) {
             var url = '/api/v1/crates/' + data.crate.get('name') + '/following';
-            ajax(url).then(function(d) {
-                controller.set('following', d.following);
-            }).finally(function() {
-                controller.set('fetchingFollowing', false);
-            });
+
+            ajax(url).
```

Comment:
Could this put the period at the start of the next lines? I tend to prefer the style like:

```js
ajax(foo)
    .bar(...)
    .baz(...)
```